workspaces/ce-workspace/scripts/create-host-certificate.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
This is in shared/scripts/ I think this one is not even used

Reply:
Removed the file.